kv/txn_coord_sender.go

```diff
@@ -504,8 +503,8 @@ func (tc *TxnCoordSender) sendOne(ctx context.Context, call proto.Call) {
 		// without starting a new call, which is hard to trace. Plus, the
 		// below depends on default configuration.
 		tmpDB, err := client.Open(
-			fmt.Sprintf("//%s?priority=%d",
-				call.Args.Header().User, call.Args.Header().GetUserPriority()),
+			fmt.Sprintf("//?priority=%d",
```

Comment:
ah, yes. this one did nothing.
We have 9 instances of `//root@` in the changes I just made, plus the one just highlighted here without the `@` which does nothing.
I don't mind much in tests, but this is also specified in server.go and node.go. Those should really be using proper struct initializers rather than a short-hand url.